<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admArea</class>
 <widget class="QDialog" name="admArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="tbEdicionA">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edicion</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbAlta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Alta</string>
   </property>
  </widget>
  <widget class="QWidget" name="edicion" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Matricula</string>
    </property>
   </widget>
   <widget class="QLabel" name="cMatricula">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cMatricula</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cNombre">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cAP">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cAM">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Materno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cCorreo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Celular</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cCelular">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLabel" name="cEdad">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cEdad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lEmpleados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Clave</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Area</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="alta" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="tMatricula">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Matricula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tApaterno">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tAmaterno">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Materno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sEdad">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tClave">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tClaveC">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Confirmación)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmar">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>250</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tCorreo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Celular:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tCelular">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="tbCursos">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cursos</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbProfesores">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Profesores</string>
   </property>
  </widget>
  <widget class="QWidget" name="altaCursos" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="lCursos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>ID Curso</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Locación</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Profesor Asignado</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lCurso">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="lidCurso">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matricula Profesor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lmProfesor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lnomProfesor">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion del Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horario Inicio</string>
    </property>
   </widget>
   <widget class="QLabel" name="lHorario">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hora Final</string>
    </property>
   </widget>
   <widget class="QLabel" name="lHorarioF">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seccion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSeccion">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Creacion Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Curso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cNombreCurso">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbArea">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Deportes de Combate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Escuela de Gimnasia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Centro Acuatico</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Curso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cIdCurso">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>614</x>
      <y>250</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="tbCreacionCurso">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Alta Cursos</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbGestionarCursos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestionar Recursos</string>
   </property>
  </widget>
  <widget class="QWidget" name="gestionCursos" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="cursosP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>ID Curso</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Profesor Asignado</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lCurso_19">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_21">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informacion del Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_22">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horario</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_23">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Locación</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurso_24">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seccion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ctIdCurso">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSeccion">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLocaciones">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHorarios">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="aceptarGR">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMatriculas">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>alta</zorder>
  <zorder>edicion</zorder>
  <zorder>gestionCursos</zorder>
  <zorder>altaCursos</zorder>
  <zorder>tbEdicionA</zorder>
  <zorder>tbAlta</zorder>
  <zorder>tbCursos</zorder>
  <zorder>tbProfesores</zorder>
  <zorder>tbCreacionCurso</zorder>
  <zorder>tbGestionarCursos</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
